sdk/endpoint/composite/connection.go

```diff
@@ -87,6 +88,14 @@ func (cce *ConnectionCompositeEndpoint) Close(ctx context.Context, connection *c
 	return &empty.Empty{}, nil
 }
 
+func (cce *ConnectionCompositeEndpoint) generateIfName() string {
+	ifName := "nsm" + cce.id.MustGenerate()
```

Comment:
FYI... ifNames are limited to 15 chars...

Reply:
That's fine. The UUID generated should be 10 chars.

Reply:
I figured :)  I just know it drove me nuts when I hit it (which is why we now check in the dataplane and issue a super super clear error if you try to go to big :) ).

Reply:
Ah... I misread... got it :)